qiskit/qobj/converters/pulse_instruction.py

```diff
@@ -335,6 +335,8 @@ def convert_frame_change(self, instruction):
             instruction (PulseQobjInstruction): frame change qobj
         Returns:
             Schedule: Converted and scheduled Instruction
+        Raises:
```

Comment:
Can we remove this raises since the check has been removed now?

Reply:
Sorry! I overlooked. thanks.